drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/cloned/feel/BuiltInType.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.scenariosimulation.kogito.client.cloned.feel;
```

Comment:
What is the plan about the `org.drools.workbench.screens.scenariosimulation.kogito.client.cloned`.

Just copying the source from other existing module is not good approach.

Reply:
@jomarko 
We discussed this with @danielezonca and @tarilabs  - sooner or later a common "client-side" solution has to be implemented to be used by both scesim and DMN.

Reply:
I am not sure why I am involved for "client-side"